hadoop-hdds/common/src/test/java/org/apache/hadoop/ozone/upgrade/InjectedUpgradeFinalizationExecutor.java

```diff
@@ -66,38 +65,34 @@ public int getValue() {
   }
 
   @Override
-  public Void execute(Storage storageConfig,
-                      BasicUpgradeFinalizer basicUpgradeFinalizer)
-      throws Exception {
+  public Void execute(T component, BasicUpgradeFinalizer finalizer)
```

Comment:
this actually can be lowercase void, we always return null from here and from the ancestor as well, and we are not implementing anything here that requires a return type.

Reply:
Fixed